cmd/harmony/main.go

```diff
@@ -101,6 +102,7 @@ var (
 	enableMemProfiling = flag.Bool("enableMemProfiling", false, "Enable memsize logging.")
 	enableGC           = flag.Bool("enableGC", true, "Enable calling garbage collector manually .")
 	blsKeyFile         = flag.String("blskey_file", "", "The encrypted file of bls serialized private key by passphrase.")
+	blsFolder          = flag.String("blsfolder", ".hmy/blskeys", "The folder that stores the bls keys; same blspass is used to decrypt all bls keys; all bls keys mapped to same shard")
```

Comment:
hmy is a unix world binary, please use directory, not folder

Reply:
need to check with @LeoHChen if he has assumed default to be `.hmy/blskeys` for the current mainnet upgrade.

Reply:
not changing for now as @LeoHChen deploy scripts hard code this.